tensorboardX/summary.py

```diff
@@ -123,13 +150,13 @@ def make_histogram(values, bins):
     # void Histogram::EncodeToProto in histogram.cc
     for i, c in enumerate(counts):
         if c > 0:
+            start = i - 1
```

Comment:
Just noticed that if the loop breaks at i==0, start=-1 (also happens in the original code), which don't make sense.

Reply:
max(start, 0)?